Student/Student/Submissions/SubmissionDetails/SubmissionDetailsViewController.swift

```diff
@@ -67,19 +67,17 @@ class SubmissionDetailsViewController: UIViewController, SubmissionDetailsViewPr
 
         NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow(_:)), name: UIResponder.keyboardWillShowNotification, object: nil)
 
+        guard
```

Comment:
I think an `if` would be better here. The duplicated `presenter?.viewIsReady()` line with the `return` makes the logic more complex for me.

Reply:
Good idea!